lib/rubocop/cop/rails/env_local.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Checks for usage of `Rails.env.development? || Rails.env.test?` which
+      # can be replaced with `Rails.env.local?`, introduced in Rails 7.1.
+      #
+      # @example
+      #
+      #   # bad
+      #   Rails.env.development? || Rails.env.test?
+      #
+      #   # good
+      #   Rails.env.local?
+      #
+      class EnvLocal < Base
+        extend AutoCorrector
+        extend TargetRailsVersion
+
+        MSG = 'Use `Rails.env.local?` instead.'
+        LOCAL_ENVIRONMENTS = %i[development? test?].to_set.freeze
+
+        minimum_target_rails_version 7.1
+
+        # @!method rails_env_local_candidate?(node)
+        def_node_matcher :rails_env_local_candidate?, <<~PATTERN
+          (or
+            (send (send (const {cbase nil? } :Rails) :env) $%LOCAL_ENVIRONMENTS)
+            (send (send (const {cbase nil? } :Rails) :env) $%LOCAL_ENVIRONMENTS)
+          )
+        PATTERN
+
+        def on_or(node)
+          rails_env_local_candidate?(node) do |*environments|
+            next unless environments.to_set == LOCAL_ENVIRONMENTS
```

Comment:
Can you resolve the following offense?
https://app.circleci.com/pipelines/github/rubocop/rubocop-rails/2710/workflows/7b3e5571-a8f9-460f-8fef-eecf228283e3/jobs/18160

Reply:
Done! You're too fast for me! 😅